multidict/_multidict.c

```diff
@@ -709,13 +709,21 @@ static inline void
 multidict_tp_dealloc(MultiDictObject *self)
 {
     PyObject_GC_UnTrack(self);
+#if PY_MAJOR_VERSION >= 3 && PY_MINOR_VERSION >= 9
+    Py_TRASHCAN_BEGIN(self, multidict_tp_dealloc)
```

Comment:
I'm not very familiar with these helpers, but shouldn't the C instructions be delimited with a semicolon?
```suggestion
    Py_TRASHCAN_BEGIN(self, multidict_tp_dealloc);
```

Reply:
This one doesn't end a semicolon, but is an loop opening:
https://github.com/python/cpython/blob/b5dc0f83adda0bde1b176c3a84cc31cd94254e73/Include/cpython/object.h#L480-L482
https://github.com/python/cpython/blob/b5dc0f83adda0bde1b176c3a84cc31cd94254e73/Include/cpython/object.h#L462-L473

So I guess we're good here... I'll mark this thread as resolved.